src/algebra/quaternion.lean

```diff
@@ -573,3 +583,55 @@ monoid_with_zero_hom.map_div norm_sq a b
 end field
 
 end quaternion
+
+private theorem one_le_four : 1 ≤ 4 := by simp
+
+namespace cardinal
+
+open_locale cardinal quaternion
+
+section quaternion_algebra
+
+variables {R : Type*} (c₁ c₂ : R)
+
+private theorem pow_four [infinite R] : #R ^ 4 = #R :=
```

Comment:
Can you generalize this to lemmas about `pow_bit0` and `pow_bit1`? Then they might be useful simp-lemmas in a file on cardinals.

Reply:
Sure, I'll put that on another PR.

Reply:
Actually, I think I'd rather leave this as is. Seems kind of dumb to add lemmas about `#α ^ (bit0 n)` and such when the corresponding lemmas already exist for general cardinals. Also, directly using that `1 ≤ 4` leads to a much quicker proof in this particular circumstance.